bigframes/series.py

```diff
@@ -680,6 +680,17 @@ def rdivmod(self, other) -> Tuple[Series, Series]:  # type: ignore
         return (self.rfloordiv(other), self.rmod(other))
 
     def __matmul__(self, other):
```

Comment:
Can we add `__rmatmul__`?

Reply:
Let me take that up as a follow up. Seems there is transpose involved so I need to understand the behavior better b/313748747